pkg/controller/datadogagent/clusteragent.go

```diff
@@ -342,6 +429,10 @@ func newClusterAgentPodTemplate(agentdeployment *datadoghqv1alpha1.DatadogAgent,
 	return newPodTemplate
 }
 
+func getClusterAgentCustomConfigConfigMapName(dda *datadoghqv1alpha1.DatadogAgent) string {
+	return fmt.Sprintf("%s-cluster-datadog-yaml", dda.Name)
```

Comment:
```suggestion
	return fmt.Sprintf("%s-cluster-datadog.yaml", dda.Name)
```
I'm I missing something?

Reply:
it the name of the configmap,  not the file